src/cmd/initialize.go

```diff
@@ -44,46 +50,55 @@ var initCmd = &cobra.Command{
 		zarfLogo := message.GetLogo()
 		_, _ = fmt.Fprintln(os.Stderr, zarfLogo)
 
-		err := validateInitFlags()
-		if err != nil {
+		if err := validateInitFlags(); err != nil {
 			message.Fatal(err, "Invalid command flags were provided.")
```

Comment:
As discussed (in person-ish), this PR will set the baseline for lang file capability and we'll slowly go about utilizing the lang file as we complete future work instead of also trying to cram that into this PR.